packages/device-id/package.json

```diff
@@ -23,14 +23,19 @@
   ],
   "license": "Apache-2.0",
   "main": "dist/index.js",
+  "types": "./dist/index.d.ts",
   "exports": {
     ".": {
-      "require": "./dist/index.js",
-      "import": "./dist/.esm-wrapper.mjs",
-      "types": "./dist/index.d.ts"
+      "require": {
+        "default": "./dist/index.js",
+        "types": "./dist/index.d.ts"
+      },
+      "import": {
+        "default": "./dist/.esm-wrapper.mjs",
+        "types": "./dist/index.d.ts"
```

Comment:
This comment [here](https://github.com/microsoft/TypeScript/issues/52363#issuecomment-1659227177) says that you need separate types files for require/import but so far in my testing by just npm linking things together it seems to work?

What I'll do once this PR lands is to make sure we bump all these packages where they are used so that I find and fix any errors and not someone else.